crates/bevy_render/src/camera/camera.rs

```diff
@@ -477,7 +477,7 @@ impl NormalizedRenderTarget {
                         window.resolution.physical_width(),
                         window.resolution.physical_height(),
                     ),
-                    scale_factor: window.resolution.scale_factor(),
+                    scale_factor: window.resolution.scale_factor() as f64,
```

Comment:
I expect we'd want to change this RenderTargetInfo field to also use an f32

Reply:
Yes you are right. Will fix that.